### ISSUE TITLE
Open VSX Registry: Publish Back to Version 0.2.384

### ISSUE THREAD

**Issue by RodogInfinite** [rust-analyzer/rust-analyzer#7960]
I'm currently working on getting mutual support between Rust-Analyzer and Theia IDE (https://github.com/eclipse-theia/theia/issues/9136, https://github.com/rust-analyzer/rust-analyzer/issues/7905). One of the missing pieces is getting rust-analyzer to be searchable within Theia's Open VSX Registry search feature, see figures below. The latest vscode engine currently supported by Theia is `1.50.0`; therefore, the published versions of Rust-Analyzer on the Open VSX Registry aren't searchable because their supported versions are above that. It would be very useful to have all versions back to 0.2.384 published so that Rust-Analyzer can be immediately searchable and will continue to be so as Theia's support for newer versions moves forward. Once the action of publishing them is performed, there should be minimum maintenance requirements unless others using older versions of Theia request specific versions to be published - which is unlikely. This fits with the ongoing effort to support publishing to the Open VSX Registry (https://github.com/rust-analyzer/rust-analyzer/issues/7284)

![image](https://user-images.githubusercontent.com/22568404/110532786-477d5c80-80eb-11eb-94c5-bce21891d464.png)

![image](https://user-images.githubusercontent.com/22568404/110532886-667bee80-80eb-11eb-8918-0aa3ba0c882f.png)

**Comment by lnicola** [comment]
I don't think we can easily acquire that version, which is pretty old, and we weren't in control of the OVSX namespace when you opened this issue.

In the meanwhile, we've gone through some pretty big changes, and can't meaningfully support Theia as long as it lags so much behind (Theia is on `1.53.2` now, we currently require `1.66`).
